parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -354,8 +354,9 @@ try
                 -MasterSubnet $global:MasterSubnet `
                 -KubeServiceCIDR $global:KubeServiceCIDR `
                 -VNetCIDR $global:VNetCIDR `

```

Comment:
I think you need to capture the backtick at the end of this line too

Reply:
Can you add a blank line after {{end}} too?

the backtick means the command is continuted on to the next line and in the non-azure stack case this could get interpreted as the if statement geting considered part of the Set-AzureCNIConfig call resulting in everything getting confused

Reply:
So the backtick continues the statement on the next line? I could just make the if statement inline, after the `-VNetCIDR` option as well